doc/src/guide/http.asciidoc

```diff
@@ -222,10 +222,24 @@ gun:request(ConnPid, "TRACE", "/", [
 	{<<"max-forwards">>, "30"}
 ]).
 
+=== Request options
+
+==== Redirecting responses to a different process
+
+By default, all data received from the server is sent to
+the owner process as a message. The behaviour can be changed
+by specifying the pid of a process to redirect response handling to.
+All request functions have an additional argument for that purpose.
+
+.GET "/organizations/ninenines" specifying a process to reply to
+
+[source,erlang]
+ReqOpts = #{reply_to => HandlerPid},
+StreamRef = gun:get(ConnPid, "/organizations/ninenines", [], ReqOpts).
+
 === Processing responses
 
-All data received from the server is sent to the owner
-process as a message. First a `gun_response` message is sent,
+First a `gun_response` message is sent,
```

Comment:
I would keep this as-is, and put the `Redirecting responses to a different process` part as its own section at the end of the document instead. It's a special case, most people won't care.